pom.xml

```diff
@@ -175,6 +186,12 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-yaml</artifactId>
+            <version>2.4.3</version>
+            <scope>test</scope>
```

Comment:
@slavaschmidt Do we need scope of `compile` for `jackson-dataformat-yaml`, or is it optional?